diff --git a/gluten-core/src/main/scala/org/apache/gluten/config/GlutenCoreConfig.scala b/gluten-core/src/main/scala/org/apache/gluten/config/GlutenCoreConfig.scala
index eb0839677..a5e3f2e1b 100644
--- a/gluten-core/src/main/scala/org/apache/gluten/config/GlutenCoreConfig.scala
+++ b/gluten-core/src/main/scala/org/apache/gluten/config/GlutenCoreConfig.scala
@@ -84,7 +84,7 @@ object GlutenCoreConfig extends ConfigRegistry {
         "Whether to enable gluten. Default value is true. Just an experimental property." +
           " Recommend to enable/disable Gluten through the setting for spark.plugins.")
       .booleanConf
-      .createWithDefault(true)
+      .createWithDefault(false)
 
   val GLUTEN_COST_MODEL =
     buildConf("spark.gluten.costModel")
diff --git a/gluten-substrait/src/main/scala/org/apache/gluten/config/GlutenConfig.scala b/gluten-substrait/src/main/scala/org/apache/gluten/config/GlutenConfig.scala
index 14610868e..346de63f1 100644
--- a/gluten-substrait/src/main/scala/org/apache/gluten/config/GlutenConfig.scala
+++ b/gluten-substrait/src/main/scala/org/apache/gluten/config/GlutenConfig.scala
@@ -1028,7 +1028,7 @@ object GlutenConfig extends ConfigRegistry {
         "The threshold to determine whether to use sort-based columnar shuffle. Sort-based " +
           "shuffle will be used if the number of partitions is greater than this threshold.")
       .intConf
-      .createWithDefault(4000)
+      .createWithDefault(0)
 
   val COLUMNAR_SHUFFLE_SORT_COLUMNS_THRESHOLD =
     buildConf("spark.gluten.sql.columnar.shuffle.sort.columns.threshold")
@@ -1036,7 +1036,7 @@ object GlutenConfig extends ConfigRegistry {
         "The threshold to determine whether to use sort-based columnar shuffle. Sort-based " +
           "shuffle will be used if the number of columns is greater than this threshold.")
       .intConf
-      .createWithDefault(100000)
+      .createWithDefault(0)
 
   val COLUMNAR_TABLE_CACHE_ENABLED =
     buildStaticConf("spark.gluten.sql.columnar.tableCache")
@@ -1594,7 +1594,7 @@ object GlutenConfig extends ConfigRegistry {
     buildStaticConf("spark.gluten.storage.hdfsViewfs.enabled")
       .doc("If enabled, gluten will convert the viewfs path to hdfs path in scala side")
       .booleanConf
-      .createWithDefault(false)
+      .createWithDefault(true)
 
   val AUTO_ADJUST_STAGE_RESOURCE_PROFILE_ENABLED =
     buildConf("spark.gluten.auto.adjustStageResource.enabled")
